conformance/src/test/scala/za/co/absa/enceladus/conformance/interpreter/InterpreterSuite.scala

```diff
@@ -39,7 +40,7 @@ class InterpreterSuite extends FunSuite with SparkTestBase {
     spark.sessionState.conf.setConfString("co.za.absa.enceladus.confTest", "hello :)")
 
     implicit val dao: EnceladusDAO = mock(classOf[EnceladusDAO])
-    implicit val progArgs = CmdConfig(reportDate = "2017-11-01")
+    implicit val progArgs = CmdConfig(reportDate = "2017-11-01", experimentalMappingRule = true)
```

Comment:
Could we also have a test case for `experimentalMappingRule = false`?

Reply:
Agree